components/ai_chat/resources/page/components/header/index.tsx

```diff
@@ -35,22 +36,20 @@ export const ConversationHeader = React.forwardRef(function (props: FeatureButto
   const shouldDisplayEraseAction = !aiChatContext.isStandalone &&
     conversationContext.conversationHistory.length >= 1
 
-  const newConversation = () => {
-    aiChatContext.onNewConversation()
-  }
-
   const activeConversation = aiChatContext.visibleConversations.find(c => c.uuid === conversationContext.conversationUuid)
-  const showTitle = (!aiChatContext.isDefaultConversation || aiChatContext.isStandalone)
+  const conversationId = useSelectedConversation()
+  const isDefault = conversationId === 'default'
+  const showTitle = !isDefault || aiChatContext.isStandalone
   const canShowFullScreenButton = aiChatContext.isHistoryEnabled && !aiChatContext.isMobile && !aiChatContext.isStandalone && conversationContext.conversationUuid
 
   return (
     <div className={styles.header} ref={ref}>
       {showTitle ? (
-        <div className={styles.conversationTitle}>
-          {!aiChatContext.isStandalone && <Button
+        <div className={styles.pageTitle}>
+          {!isDefault && !aiChatContext.isStandalone && <Button
             kind='plain-faint'
             fab
-            onClick={() => { aiChatContext.onSelectConversationUuid(undefined) }}
+            onClick={() => location.href = "/default"}
```

Comment:
If we do have a sensible context responsible for url management, perhaps we should not have to repeat the url paths, and even make functions, e.g. `AIChatRouteContext.urlForTabConversation` 🤷

Reply:
I just exported a string for now.